tests/conftest.py

```diff
@@ -65,6 +65,7 @@ def pytest_terminal_summary(terminalreporter, exitstatus, config):
 
 logging.basicConfig()
 
+EDGE_CASES = {"IdP": "Idp", "AuthNMapping": "AuthnMapping", "AuthN ": "Authn ", "IoT": "Iot"}
```

Comment:
could we have it as a json file that is shared with generator?

Reply:
Do you have a preferred location? Probably somewhere in .generator right?

Reply:
yes .generator seems appropriate